.gitlab-ci.yml

```diff
@@ -41,22 +41,7 @@ variables:
   before_script:
     - rustup show
     - cargo --version
-    - SCCACHE_ERROR_LOG=/builds/parity/parity-ethereum/sccache_debug.log
-        RUST_LOG=sccache=debug
-        sccache --start-server
-    - sccache -s
-  after_script:
-    # sccache debug info 
-    - if test -e sccache_debug.log;
-      then
-        echo "_____All crate-types:_____";
-        grep 'parse_arguments.*--crate-type' sccache_debug.log | sed -re 's/.*"--crate-type", "([^"]+)".*/\1/' | sort | uniq -c;
-        echo "_____Non-cacheable reasons:_____";
-        grep CannotCache sccache_debug.log | sed -re 's/.*CannotCache\((.+)\).*/\1/' | sort | uniq -c;
-      else
-        echo "_____No logs from sccache_____";
-        exit 0;
-      fi
+    - SCCACHE_REDIS=redis://:${REDIS}@172.17.0.1/0 sccache --start-server
```

Comment:
Is it wise to use the explicit ip address here?

Reply:
That's a default gateway address of a Docker container-internal network (so in practice it's a mapping of host's `127.0.0.1` into a local subnet).
I guess it's safe enough, provided that we're not going to mess with Docker's networking too much in this setup.

As far as I understood from my chat with @General-Beck earlier today, there's no easy way to teach `sccache` to identify that IP by hostname, and there's no _non-hacky_ lookup flow to be used instead either.